config/unicorn.rb

```diff
@@ -0,0 +1,22 @@
+worker_processes Integer(ENV["WEB_CONCURRENCY"] || 2)
+timeout 20
+preload_app true
+
+before_fork do |server, worker|
```

Comment:
Unused block argument - `server`. You can omit all the arguments if you don't care about them.<br>Unused block argument - `worker`. You can omit all the arguments if you don't care about them.